model_analyzer/record/metrics_manager.py

```diff
@@ -307,7 +307,7 @@ def _get_perf_analyzer_metrics(self,
 
         status = perf_analyzer.run(self._perf_metrics, env=perf_analyzer_env)
 
-        if perf_output_writer:
+        if perf_output_writer and perf_analyzer.output():
```

Comment:
I think it would still good to write the perf analyzer command to the output so that users can use it later to debug the issue.

Reply:
moved this condition lower 👍